delfin/tests/e2e/GetResources.robot

```diff
@@ -0,0 +1,123 @@
+*** Settings ***
+Documentation    Tests to verify that GET of resources
+
+Library                 RequestsLibrary
+Library                 Collections
+Library                 JSONLibrary
+Library                 OperatingSystem
+
+Suite Setup             Open Application
+Suite Teardown          Close Application
+
+*** Variables ***
+${delfin_url}           http://localhost:8190/v1
+${storage_pools}        storage-pools
+@{res_urls}             storage-pools  volumes  controllers  disks  ports  quotas  qtrees  filesystems  shares
+@{res_indx}             storage_pools  volumes  controllers  disks  ports  quotas  qtrees  filesystems  shares
+
+*** Test Cases ***
+GET Resources when test storage is registered
+    [Tags]    DELFIN
+
+    FOR     ${res_url}   ${res_ind}   IN ZIP    ${res_urls}   ${res_indx}
+            ${ret_json}=            Get All Resource Of     ${res_url}
+            ${res_s}=               Get Value From Json	    ${ret_json}         $..${res_ind}
```

Comment:
How about some test cases for just validating the structure of response, not the value.. This will help us to validate any real back end or simulated backend. Since values are dynamic, value validation can be specific to Test driver alone!.

Reply:
Created issue #584 for creating a software simulator for end to end tests without real hardware